src/app/ui/cli.js

```diff
@@ -124,10 +121,19 @@ module.exports = class CLI {
   }
 
   static onclick(e) {
-    CLI.HELP_UI_OPEN = !CLI.HELP_UI_OPEN
+    Session.focused().state._modals.commands = !Session.focused()
+      .state._modals.commands
   }
 
   view({ attrs }) {
+    let commandsModalOpen = false
+    if (typeof Session.focused() !== "undefined") {
+      if (
+        Session.focused().state._modals.commands === true
```

Comment:
This should use `Lens.get` and remove all the `if` statements and use a `const`

Reply:
I was trying to use optional chaining but that didn't seem to be working. `Lens` is clearly the way here, I'll look into getting that in as soon as I can.

Reply:
ok another crack at that is in